hw/top_earlgrey/dv/verilator/chip_sim.core

```diff
@@ -105,7 +105,9 @@ targets:
           - '--trace-max-array 1024'
           - '--unroll-count 512'
           - '-CFLAGS "-std=c++11 -Wall -DVM_TRACE_FMT_FST -DVL_USER_STOP -DTOPLEVEL_NAME=chip_sim_tb"'
+          - '-CFLAGS "$(CFLAGS_FOR_BUILD)" -D__STDC_CONSTANT_MACROS'
```

Comment:
Variables CFLAGS_FOR_BUILD and LDFLAGS_FOR_BUILD are chosen as this is what we use at WD for holding non-standard include paths for passing these with meson.  (In the case of meson/ninja builds we have to add custom paths and link-time parameters for using glibc-2.17 on RHEL6.)

Reply:
In this case we require this to point to `libelf` which for us is installed in a nonstandard location.
We could however add instead a call to pkg-config here.   Any preferences?

Reply:
Instead of using `__STDC_CONSTANT_MACROS` it's probably better to remove UINT64_C in https://github.com/lowRISC/opentitan/blob/b9788b8ce1bf206f9d79c659b146af8cb63099f2/hw/ip/prim/dv/prim_prince/crypto_dpi_prince/prince_ref.h. Can you give that a try (or otherwise I'll have a look.)?

Reply:
I can try this sure.  If this goes wrong would `ninja -C build-out test` find it?

Reply:
No, the prim_prince DPI code is used by DV only. (private) CI should find it, or you can manually run a regression with dvsim. But UINT64_C is really rather useless in prince_ref.h, so you can simply remove it and everything should be OK.

Reply:
I'll just remove it then.  (FYI: @udinator, @rswarbrick, @msfschaffner as maintainers of this block)